tests/io/v3/vm/editor/test_editor_api.py

```diff
@@ -0,0 +1,158 @@
+'''
+Test Editor APIs
+'''
+from pathlib import Path
+
+import requests
+import responses
+
+from tenable.io.v3.base.iterators.explore_iterator import (CSVChunkIterator,
+                                                           SearchIterator)
+
+from .objects import (CONFIGURATION_DETAILS, CONFIGURATION_ID, ETYPE,
```

Comment:
here you can simply use `from .objects import *`

Reply:
absolute path fixed. star import will cause flake8 failures